include/z64.h

```diff
@@ -1167,21 +1167,21 @@ struct FireObj {
 }; // size = 0x8B
 
 typedef struct {
-    /* 0x00 */ u8   seqIndex;
-    /* 0x01 */ u8   nightSeqIndex;
-    /* 0x02 */ u8   unk_02;
+    /* 0x0 */ u8   seqIndex;
+    /* 0x1 */ u8   nightSeqIndex;
+    /* 0x2 */ u8   unk_02;
 } SoundContext; // size = 0x3
 
 typedef struct {
-    /* 0x00 */ s32 enabled;
-    /* 0x04 */ s32 timer;
+    /* 0x0 */ s32 enabled;
+    /* 0x4 */ s32 timer;
 } FrameAdvanceContext; // size = 0x8
 
 typedef enum {
-    /* 00 */ GAMEOVER_INACTIVE,
-    /* 01 */ GAMEOVER_DEATH_START,
-    /* 02 */ GAMEOVER_DEATH_WAIT_GROUND,    // wait for player to fall and hit the ground
-    /* 03 */ GAMEOVER_DEATH_FADE_OUT,       // wait before fading out
+    /*  0 */ GAMEOVER_INACTIVE,
+    /*  1 */ GAMEOVER_DEATH_START,
+    /*  2 */ GAMEOVER_DEATH_WAIT_GROUND,    // wait for player to fall and hit the ground
+    /*  3 */ GAMEOVER_DEATH_FADE_OUT,       // wait before fading out
 
     /* 20 */ GAMEOVER_REVIVE_START = 20,
```

Comment:
```suggestion
    /*  3 */ GAMEOVER_DEATH_FADE_OUT,       // wait before fading out
    /* 20 */ GAMEOVER_REVIVE_START = 20,
```
Having an extra space in here looks weird to me? Even if it does separate out two sections of the enum. Thoughts?